src/App.ts

```diff
@@ -1,12 +1,42 @@
 import express from 'express';
 import path from 'path';
+import {User} from "./backend/Models";
+import bodyParser from 'body-parser';
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 3000;
+app.use(bodyParser.json());
+
+// mongoose connection, connect to local database with db name "project", creates new database if "project" doesn't exist
+mongoose.connect('mongodb://localhost:27017/project', {useNewUrlParser: true});
```

Comment:
Maybe we can make this configureable so we can use a separate database in tests?

Reply:
Yep sure